web/src/components/WelcomeModal.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalBody,
+  ModalFooter,
+  Button,
+} from "@chakra-ui/react";
+import { useState } from "react";
+import LAPTOP_CIRCLE_ICON from "../assets/laptop-circle-icon.svg";
+
+export function WelcomeModal() {
+  const [openModal, setOpenModal] = useState(true);
+
+  function handleCloseModal() {
+    setOpenModal(false);
+  }
+
+  return (
+    <Modal isOpen={openModal} onClose={handleCloseModal} size="2xl" isCentered>
```

Comment:
I'm not a huge fan of this modal. I already have an explanation of what the demo is which shows up on the right side of the map. If you feel that something more "in the user's face" is needed, I can come up with something, but it should be in a separate PR and probably involve the design team.

Reply:
This is entirely planned by the design team. You can see more details here https://www.figma.com/file/uFWFj9AtRHmbpnpMjPVGxJ/Real-time-dashboard-application?node-id=3255%3A34170
I can remove the description of the right side section as it's planned in the mockups. And I agree it should be done in a separate PR, will open a new one